inspirehep/modules/records/receivers.py

```diff
@@ -67,6 +67,27 @@ def enhance_record(sender, json, *args, **kwargs):
     populate_abstract_source_suggest(sender, json, *args, **kwargs)
     populate_title_suggest(sender, json, *args, **kwargs)
     after_record_enhanced.send(json)
+    add_book_autocomplete(sender, json, *args, **kwargs)
+    after_record_enhanced.send(json)
+
+
+def add_book_autocomplete(sender, json, *args, **kwargs):
+    if 'book' in force_list(json.get('document_type')):
+        result = json.get('bookautocomplete', [])
+        result.extend(force_list(get_value(json, 'titles.title')))
+        result.extend(force_list(get_value(json, 'authors.full_name')))
+        titles = force_list(get_value(json, 'titles.title'))
```

Comment:
The purpose of this is to add these values to the autocompletion but in the payload as well so that I can fill the fields afterwards.